addons/website/models/mixins.py

```diff
@@ -183,7 +183,7 @@ class WebsitePublishedMixin(models.AbstractModel):
     _description = 'Website Published Mixin'
 
     website_published = fields.Boolean('Visible on current website', related='is_published', readonly=False)
-    is_published = fields.Boolean('Is Published', copy=False, default=lambda self: self._default_is_published(), index=True)
+    is_published = fields.Boolean('Is Published', copy=False, default=lambda self: self._default_is_published())
```

Comment:
Are you sure for this one too? There was a performance fix about this field: https://github.com/odoo/odoo/pull/130734/files
Maybe `index.true` ?